README.md

```diff
@@ -17,6 +17,31 @@ Consider an Flight Booking Application as shown in the figure. It consists of a
 
 Tracking down performance issues in a dynamic microservices environment can be challenging and more often than not, stack or runtime specific optimizations are written off as too complex. This is because runtime optimization is a very involved effort and requires deep expertise. Common fixes are mostly limited to increasing pod resources, fixing application logic to make it more optimal or increasing horizontal pod auto-scaling. 
 
+##Usecases
```

Comment:
There should be space between the ## and the text so that it gets rendered as heading